ext/node_resolver/analyze.rs

```diff
@@ -196,7 +196,7 @@ impl<TCjsCodeAnalyzer: CjsCodeAnalyzer, TNodeResolverEnv: NodeResolverEnv>
             &referrer,
             // FIXME(bartlomieju): check if these conditions are okay, probably
             // should be `deno-require`, because `deno` is already used in `esm_resolver.rs`
-            &["deno", "require", "default"],
+            &["deno", "node", "require", "default"],
```

Comment:
What order does node do here? Should this be "require", "node" since "node" can possibly be ESM? (the "deno" condition should probably go after "require" too)

Reply:
The order of conditions here doesn't matter, the order only matters in the exports declaration in package.json. You match them in order that they're declared, and you're just checking that the condition is present.

Reply:
Ah, yes that's correct.